src/Camera_TEST.cc

```diff
@@ -178,6 +178,11 @@ void CameraTest::RenderTexture(const std::string &_renderEngine)
   camera->SetImageHeight(80u);
   EXPECT_EQ(80u, camera->ImageHeight());
 
+  double width = 100
+  camera.SetImageWidth(width)
+  double aspectRatio = width / camera.ImageHeight()
+  EXPECT_NEAR(aspectRatio, camera.AspectRatio(), 1e-6)
```

Comment:
This is missing `;` at the end of each line and `camera` is a pointer. This test fails though, you will need to try to investigate why and how to correct this

Since you are using `ogre` on your system, update `ogre2` to `ogre` here: https://github.com/gazebosim/gz-rendering/blob/a3dc1aeb8102c1e96616dfee06310c226c1b995c/test/test_config.h.in#L16

To compile the test, run  `colcon build --cmake-args -DBUILD_TESTING=ON --merge-install --packages-select ignition-rendering6` in the root of your workspace. To run the test you can do `./build/ignition-rendering6/bin/UNIT_Camera_Test`

Reply:
You should also add a similar test with `SetImageHeight`